recommendation/src/main/java/com/yas/recommendation/kafka/config/consumer/ProductCdcKafkaListenerConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.yas.recommendation.kafka.config.consumer;
+
+import com.yas.commonlibrary.kafka.cdc.config.BaseKafkaListenerConfig;
+import com.yas.commonlibrary.kafka.cdc.message.ProductCdcMessage;
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ * Product CDC kafka listener, support convert product cdc message to java object.
+ */
+@EnableKafka
+@Configuration
+@EnableScheduling
```

Comment:
Can you explain how the @EnableScheduling is being used here? I don't see any scheduled tasks defined.

Reply:
It's just redundant code, let's me remove.